drone/starlark/starlark.go

```diff
@@ -223,16 +220,14 @@ func generate(c *cli.Context) error {
 		return ioutil.WriteFile(target, buf.Bytes(), 0644)
 	}
 
-	manifest, err := yaml.Parse(buf)
-	if err != nil {
-		return err
+	yml, yErr := yaml.JSONToYAML(buf.Bytes())
+	if yErr != nil {
+		return fmt.Errorf("failed to convert to YAML: %v", yErr)
 	}
 	buf.Reset()
-	pretty.Print(buf, manifest)
+	buf = bytes.NewBuffer(yml)
```

Comment:
instead of creating a new buffer, you can probably do `buf.Write` here since we reset the existing buffer. Or if you prefer to create a new buffer, you could remove the `buf.Reset`